src/ui/statusBarActionView.ts

```diff
@@ -76,7 +76,8 @@ export default class StatusBarActionView {
       RestartDafny
     ];
     const currentDocumentStatus = this.compilationStatusView.getCurrentDocumentStatus();
-    if(currentDocumentStatus === Messages.CompilationStatus.VerificationFailed
+    if(currentDocumentStatus === Messages.CompilationStatus.VerificationFailedOld
```

Comment:
Good job on making this backward compatible. I wonder if there is a better way to not depend on these messages. No action needed here if you don't have an idea.